molecule_vagrant/playbooks/prepare.yml

```diff
@@ -12,7 +12,12 @@
         (test -e /usr/bin/yum && sudo yum -y -qq install python3) ||
         (test -e /usr/sbin/pkg && sudo env ASSUME_ALWAYS_YES=yes pkg update && sudo env ASSUME_ALWAYS_YES=yes pkg install python3) ||
         (test -e /usr/sbin/pkg_add && sudo /usr/sbin/pkg_add -U -I -x python%3.7) ||
-        echo "Warning: Python not bootstrapped due to unknown platform."
+        echo "ERROR: Python not bootstrapped due to unknown platform." && exit 234
```

Comment:
I do not think this is desirable because it will prevent people from using the module on platforms that may already have a working vagrant/python.

We should fail if is missing or failing not if we have no clue where is running.